include/libmem/libmem.h

```diff
@@ -40,10 +40,14 @@
 #endif
 
 /* Resolve import/export */
-#ifdef LM_EXPORT
-#	define LM_API LM_API_EXPORT
+#ifdef LM_STATIC
+#	define LM_API
 #else
-#	define LM_API LM_API_IMPORT
+#	ifdef LM_EXPORT
+#		define LM_API LM_API_EXPORT
+#	else
+#		define LM_API LM_API_IMPORT
+#	endif
 #endif
```

Comment:
It should rely over api.h file instead, since recent updates?

Reply:
Yes, I've just pushed an update.